src/FilterRelationalAggregatesPlugin.ts

```diff
@@ -383,7 +386,12 @@ group by true)`;
                 {
                   description: `Aggregates across related \`${foreignTableTypeName}\` match the filter criteria.`,
                   type: AggregateType,
-                  applyPlan(
+                  applyPlan: EXPORTABLE( 
+                    (
+                    PgAggregateConditionStep, 
+                    sql, 
+                    pgWhereConditionSpecListToSQL
+                    ) => function (
```

Comment:
Do you care whether the second function is an arrow function of a `function` keyword function? I've seen both.

Reply:
Not particularly. `this` isn't used in the function so it doesn't really make any difference. There used to be a performance cost for arrow functions but that's not been the case in years. Generally I go with arrow functions because they're more concise, but using traditional functions gives the option of giving the function a name which can be helpful for debugging.